### PR TITLE
Updated logits.

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Properly formatted the logits definition.
Fixes https://github.com/tensorflow/tensorflow/issues/47553

### CODE REVIEW

File: site/en/tutorials/keras/classification.ipynb [hunk 1]
```diff
@@ -585,7 +585,7 @@
         "### Make predictions\n",
         "\n",
         "With the model trained, you can use it to make predictions about some images.\n",
-        "The model's linear outputs, [logits](https://developers.google.com/machine-learning/glossary#logits). Attach a softmax layer to convert the logits to probabilities, which are easier to interpret. "
+        "Attach a softmax layer to convert the model's linear outputs, [logits](https://developers.google.com/machine-learning/glossary#logits) to probabilities, which are easier to interpret. "
```
Comment (8bitmp3) [inline site/en/tutorials/keras/classification.ipynb:588]:
```suggestion
        "Attach a softmax layer to convert the model's linear outputs—[logits](https://developers.google.com/machine-learning/glossary#logits)—to probabilities, which should be easier to interpret."
```

**Review by 8bitmp3 — APPROVED** [review]
